.github/workflows/ci.yml

```diff
@@ -466,6 +466,32 @@ jobs:
         run:  |
           cd build
           timeout --signal=SIGABRT 60m ./tst/webrtc_client_test
```

Comment:
Should we also add a macos-old-openssl build?

Reply:
I probably missed it. I will add it.